client/layout/masterbar/checkout.jsx

```diff
@@ -51,4 +51,4 @@ class CheckoutMasterbar extends React.Component {
 	}
 }
 
-export default connect( () => (null, { recordTracksEvent }) )( localize( CheckoutMasterbar ) );
+export default connect( null, { recordTracksEvent } )( localize( CheckoutMasterbar ) );
```

Comment:
Oops! Good catch, @tyxla! That should have been two functions or two objects, not a function with both 🤦